components/contract-components/contract-release-form/index.tsx

```diff
@@ -119,6 +119,11 @@ export const ContractReleaseForm: React.FC<ContractReleaseFormProps> = ({
           ?.publishedMetadata,
         changelog: "",
         version: placeholderVersion,
+        displayName:
+          prePublishMetadata.data?.latestPublishedContractMetadata
+            ?.publishedMetadata.title ||
+          prePublishMetadata.data?.preDeployMetadata.info.notice ||
+          "",
```

Comment:
this makes sense, but `notice` should really be just description

`title` -> `displayName`
`notice` -> `description`

based on the natspec at least, I assume u have a reason to do it differently here?